packages/relative-values/src/values/ModelEvaluator.ts

```diff
@@ -95,7 +95,7 @@ function buildRelativeValue({
 
 export class ModelEvaluator {
   private constructor(
-    private model: Model, // unused for now
+    public model: Model, // unused for now
```

Comment:
This PR makes it used, so the comment should be removed.

Reply:
True